cmd/wallet-web/src/pages/OIDCShare.vue

```diff
@@ -319,6 +321,15 @@ export default {
       const encodedSignature = encode(signature).slice(0, -1);
       this.idToken = `${encodedHeader}.${encodedPayload}.${encodedSignature}`;
     },
+    generateVPToken() {
+      this.vpToken = JSON.stringify({
+        '@context': this.presentations[0].verifiableCredential[0]['@context'],
+        type: ['VerifiablePresentation'],
+        id: this.presentations[0].verifiableCredential[0].id,
+        holder: 'did:example:holder',
+        proof: [],
+      });
+    },
```

Comment:
We shouldn't build this manually -  we need to use the underlying functions to build VP with proof.

Reply:
Changed and now using API